src/css/main.css

```diff
@@ -0,0 +1,55 @@
+.App {
+  width: 700px;
+  margin: 100px auto;
+}
+
+.App,
+.buttonPanel,
+.display {
+  display: flex;
+}
+
+.App,
+.buttonPanel {
+  display: flex;
+  flex-direction: column;
+}
+
+.display {
+  background-color: #808080;
+  align-items: center;
+  justify-content: flex-end;
+  color: #fff;
+  padding-right: 20px;
+  font-size: 2.5em;
+}
+
+.display,
+.row,
+.button {
+  font-family: inherit;
+  height: 100px;
+  font-weight: bold;
+}
+
+.row,
+.button {
+  font-size: 1.8em;
+}
+
+.button {
+  width: 25%;
+  background-color: orange;
+}
+
+.button:hover {
+  opacity: 0.8;
+}
+
+.wide {
+  width: 50%;
+}
```

Comment:
1. you should create a class for `orange` with `background-color: orange;` separately 
2. you should also create a class for the normal size `width: 25%;` separately, so you can dynamically choose which class you want in the app logic